core/lib/instrument.c

```diff
@@ -6799,7 +6799,7 @@ dr_get_mcontext_priv(dcontext_t *dcontext, dr_mcontext_t *dmc, priv_mcontext_t *
     else if (TEST(DR_MC_CONTROL, dmc->flags))
```

Comment:
(Putting commit title/description comment here to get an action item that can be marked Done and a separate thread.)

style nit 1: Title should have a colon after the issue number.

style nit 2: Description should have a final line that is either `Fixes #4508` or `Issue: #4508` (the former here).

style nit 3: suggestion: include the key `dr_get_mcontext` in the title, and remove the sort of implied "Dynamorio"  (should be "DynamoRIO", or "DR" for short) (it will also then fit in the github issue title); maybe sthg like `i#4508: Stolen register not loaded in dr_get_mcontext on AArch64`.

Reply:
For the description change, would you consider me force pushing a single commit with all the changes bad form. Or is it best to always do it incrementally and squash the commits before the merge.

Reply:
Never force push.  The merge is always a squash with an edited description: edit it there.  Please remove all the star lines and create a single message at that time as instructed in the wiki.